include/nod/nod.hpp

```diff
@@ -17,10 +17,8 @@ namespace nod {
 		/// Interface for type erasure when disconnecting slots
 		struct disconnector {
 			virtual void operator()( std::size_t index ) const = 0;
+			virtual ~disconnector() {}
```

Comment:
pedantic, could use `= default`
```suggestion
			virtual ~disconnector() = default;
```

Reply:
Agreed. I am willing to change that when/if I get feedback form the maintainer.